src/views/collection-view/components/collection-meta/collection-meta.module.css

```diff
@@ -17,8 +17,7 @@
 }
 
 .cta {
-	display: flex;
-	align-items: center;
+	width: 100%;
```

Comment:
The `.cta` class now handles width and margin only.

The `.ctaText` and `.ctaIcon` removed in this file are not used anymore, as they're now rendered by `CollectionMetaProgress`.